Add topics to Secret Handshake exercise

filipolszewski opened the pull request:
<!-- Your content goes here: -->

Fix #848 

First time contributing, hope this helps :)

<!-- DO NOT EDIT BELOW THIS LINE! -->
---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

FridaTveit commented on config.json:
```diff
@@ -133,7 +133,12 @@
+        "loops",
+	"transforming",
+        "conditionals",
+	"cryptography"
```
Great suggestions! I would add `enumerations` since it uses an `enum` and maybe also `mathematics` and `bitwise_operations`? What do you think?

Also, the indentation seems to have gone a bit wrong. They should all be indented two spaces in from `topics`, like `loops` and `conditionals` are :)

FridaTveit replied:
Oh and also `integers` would be a good topic :)

filipolszewski replied:
Thanks for all the informations and suggestions. I'm up for adding more topics - I wasn't sure if there's a suggested/maximum number of topics for each exercise, so I kept it minimal.

FridaTveit replied:
I don't think we have a suggested maximum or minimum number of topics at the moment. Adding topics is still a relatively new things so that might change in the future, but for the moment I think we can just add as many topics as we feel are relevant :)

filipolszewski commented:
I added all suggestions and fixed that strange indentation. Good point on 'bitwise_operations" topic - must have miseed it the first time I've checked through the topic list.
